cookbooks/creating_a_context_configuration_extension.rst

```diff
@@ -212,14 +212,77 @@ through the ``HelloWorldExtension``, ensuring it gets loaded:
       // ...
   }
 
+Finally, we can define a custom configuration object for use in a project's ``behat.php``.
+This step is optional - users can also register your extension with the generic
+``Behat\Config\Extension`` class. However, providing a custom class will make it easier
+for users to discover and configure the correct settings for your extension.
+
+.. code-block:: php
+
+  <?php
+
+  namespace HelloWorld\Config;
+
+  use Behat\Config\ExtensionConfigInterface;
+  use HelloWorld\ServiceContainer\HelloWorldExtension;
+
+  final class HelloWorldExtension implements ExtensionConfigInterface
+  {
+
+      // You don't need to set defaults here, they are defined along with the config structure in
+      // HelloWorld\ServiceContainer\HelloWorldExtension::configure(). It is important that this
+      // object only returns settings that the user has explicitly configured.
+      private array $settings = [];
+
+      // These are the methods Behat uses to read the configuration object
+
```

Comment:
In previous code in this cookbook we don't use a blank line between some comment and the related function, so I think it is best to keep the same standard here and remove these blank lines

Reply:
Ah, I now see that this comment belongs to more than one function, in this case the blank line makes sense, ignore me

Reply:
Actually that's a fair point, I've changed the grammar of those comments to make even clearer that they are referring to multiple methods.